rest-fights/src/main/java/io/quarkus/sample/superheroes/fight/rest/FightResource.java

```diff
@@ -121,4 +121,30 @@ public FightResource(FightService service) {
 	public String hello() {
 		return "Hello Fight Resource";
 	}
+
+	@GET
+	@Produces(TEXT_PLAIN)
+	@Path("/hello/heroes")
+	@Tag(name = "hello")
+	@Operation(summary = "Ping Heroes hello")
+	@APIResponse(
+		responseCode = "200",
+		description = "Ping Heroes hello"
+	)
+	public String helloHeroes() {
+		return service.helloHeroes();
+	}
+
+	@GET
+	@Produces(TEXT_PLAIN)
+	@Path("/hello/villains")
+	@Tag(name = "hello")
+	@Operation(summary = "Ping Villains hello")
+	@APIResponse(
+		responseCode = "200",
+		description = "Ping Villains hello"
+	)
+	public String helloVillains() {
```

Comment:
Should this be `@NonBlocking`?

Reply:
How would you implement the `VillainClient::helloVillains' then ?

Reply:
In `VillainClient`, add `.rx(UniInvoker.class)` to the chain, just like in `VillainClient.getRandomVillain()`.